docs/next.config.js

```diff
@@ -5,13 +5,22 @@ const semver = require('semver');
 
 const { version } = require('./package.json');
 
+const { schema_sync } = require('./scripts/schema-sync');
+
 // copy versions/v(latest version) to versions/latest
 // (Next.js only half-handles symlinks)
 const vLatest = join('pages', 'versions', `v${version}/`);
 const latest = join('pages', 'versions', 'latest/');
 removeSync(latest);
 copySync(vLatest, latest);
 
+// Starting with version 38, download xdl schema at build time
+// (and when starting a dev server)
+for (let i = 38; i <= parseInt(version, 10); i++) {
+  schema_sync(i);
+}
+schema_sync('unversioned');
```

Comment:
The `schema_sync` appears to return a promise, right? I'm not sure if this is downloading "on time" for Next to include it when it starts building?

Reply:
And I would recommend using `camelCase` instead of `snake_case`, matches the other methods better 😄

Reply:
I believe `schema_sync` fully resolves its axios.get() promises and async-ly writes to the relevant files, and does not have to return a promise, per say. 

However, I agree that I am not sure this resolving will always happen on time at the moment, esp. if when it begins to write 5 or so versions, instead of just the 2 available right now.